pwnlib/context/__init__.py

```diff
@@ -351,7 +351,7 @@ class ContextType(object):
         'os': 'linux',
         'proxy': None,
         'signed': False,
-        'terminal': None,
+        'terminal': [],
```

Comment:
This is a list instance, which may lead to weird things happening by modifying `context.terminal` in-place.  Please revert this change.